modulesets-stable/gtk-osx-python.modules

```diff
@@ -88,7 +90,7 @@
   </autotools>
 
   <autotools id="python3" autogenargs="--enable-shared" autogen-sh="configure">
-    <branch repo="python" module="3.6.1/Python-3.6.1.tar.xz" version="3.6.1">
+    <branch repo="python" module="3.6.4/Python-3.6.4.tar.xz" version="3.6.4">
       <patch file="https://git.gnome.org/browse/gtk-osx/plain/patches/Python3.4-stack_size-flag.patch" strip="1"/>
     </branch>
```

Comment:
Python is also built from tarballs in all three modulesets directories and so must be updated in all three.

Reply:
Will do.

Reply:
Done: